administrator/templates/atum/index.php

```diff
@@ -74,6 +74,8 @@
 		--template-special-color: ' . $this->params->get('special-color', '#001B4C') . ';
 	}');
 
+$app->bootComponent('com_guidedtours')->prepareAssets($app);
```

Comment:
Some nitpicking:

```suggestion
if (ComponentHelper::isEnabled('com_guidedtours')) {
    $app->bootComponent('com_guidedtours')->prepareAssets($app);
}
```

Also needs an import at the top of the file: 
```php
use Joomla\CMS\Component\ComponentHelper;
```

Reply:
I would do it a bit different https://github.com/joomla/joomla-cms/pull/40046/commits/f58b1c8790b6f0cd1e994f5545f6f1a4aaf1d248